src/fpu/fpu_instructions.h

```diff
@@ -231,8 +231,10 @@ static void FPU_FLD_I32(PhysPt addr,Bitu store_to) {
 }
 
 static void FPU_FLD_I64(PhysPt addr,Bitu store_to) {
-	constexpr int64_t int53_min = -(1LL << 53);
-	constexpr int64_t int53_max = (1LL << 53) - 1;
+	// The range of integers that can fit in the mantissa of a double.
+	// https://en.wikipedia.org/wiki/Double-precision_floating-point_format
+	constexpr int64_t mantissa_min = -(1LL << 53);
+	constexpr int64_t mantissa_max = (1LL << 53);
```

Comment:
`MantissaMax` etc. for consistency.

Reply:
man, that style is ugly for local names 😁